client/views/actions/whois.tpl

```diff
@@ -1,26 +1,35 @@
 <div>
-	<a href="#" class="user" data-name="{{whois.nickname}}">{{whois.nickname}}</a>
-	<i class="hostmask">({{whois.username}}@{{whois.hostname}})</i>:
-	<b>{{whois.realname}}</b>
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	<i class="hostmask">({{whois.user}}@{{whois.host}})</i>:
+	<b>{{whois.real_name}}</b>
 </div>
+{{#if whois.account}}
+<div>
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is logged in as <b>{{whois.account}}</b>
+</div>
+{{/if}}
 {{#if whois.channels}}
 <div>
-	<a href="#" class="user" data-name="{{whois.nickname}}">{{whois.nickname}}</a>
-	is on the following channels:
-	{{#each whois.channels}}
-		{{{parse this}}}
-	{{/each}}
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is on the following channels: {{{parse whois.channels}}}
 </div>
 {{/if}}
 {{#if whois.server}}
 <div>
-	<a href="#" class="user" data-name="{{whois.nickname}}">{{whois.nickname}}</a>
-	is connected to {{whois.server}}
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is connected to {{whois.server}} <i>({{whois.server_info}})</i>
+</div>
+{{/if}}
+{{#if whois.secure}}
+<div>
+	<a href="#" class="user" data-name="{{whois.nick}}">{{whois.nick}}</a>
+	is using a secure connection
```

Comment:
There's no security downside here, the user could as well just fire a normal IRC client and do the same `/whois` and get the information. Here, this is just displaying whatever the server reports. Many servers also sets an additional user mode to mark users as connected over SSL, so there really isn't anything to hide.